securedrop/tests/test_unit_source.py

```diff
@@ -129,6 +129,28 @@ def test_login_and_logout(self):
             self.assertIn('Sorry, that is not a recognized codename.', rv.data)
             self.assertNotIn('logged_in', session)
 
+    def test_login_with_whitespace(self):
```

Comment:
Please add a docstring to explain what this is testing and why. It would be good to reference the original issue and/or this PR.